src/pydantic_avro/avro_to_pydantic.py

```diff
@@ -1,6 +1,41 @@
 import json
 from typing import Optional, Union
 
+RESERVED_KEYWORDS= {
```

Comment:
can this list be replaced by the keyword package?
See also:
https://docs.python.org/3/library/keyword.html